right/src/ksdk_usb/usb_device_hid.c

```diff
@@ -656,10 +658,12 @@ usb_status_t USB_DeviceHidRecv(class_handle_t handle, uint8_t ep, uint8_t *buffe
     {
         return kStatus_USB_Busy;
     }
+    hidHandle->interruptOutPipeBusy = 1U;
```

Comment:
This was a typo, I meant to do this on the In pipe rather than Out pipe, but the same problem applies here.